bash_command_line_101/bash_command_line_101.md

```diff
@@ -207,6 +222,18 @@ Pictures
 Public
 ```
 
+You can click on the word "output" to collapse the output boxes. Going forward, we will omit the command prompt from the input and just show you the code to enter, like:
```

Comment:
Please also move this up on the page, it currently comes **after** the first 2 uses of "output".  Maybe line 196-ish?

```suggestion
At this point, you'll start to see output boxes that include the word "output", which show one example of the kind of output you might see when the command above is entered.  You can click on the word "output" to collapse the output boxes if you like. Going forward, we will omit the command prompt from the input and just show you the code to enter, like:
```

Reply:
Moved to immediately after the first occurrence of an output box.